### DISCUSSION TITLE
如何去替换场景模型中的材质贴图

### DISCUSSION THREAD

**Discussion by Gala-TA** [galacean/discussion#90] [Editor] [answered]
![image](https://github.com/galacean/engine/assets/35087284/71c82954-5d5c-411c-a848-7e2af7a44397)
请教一个问题？我有一个模型在场景中会大量引用，我需要对每个引用对象替换不同的材质贴图，在编辑器中要如何做到？

**Comment by luzhuang** [comment, marked as answer]
Remapped Material
![image](https://github.com/galacean/discussion/assets/10904030/c2b8e0ca-16f6-4048-aa48-19e77fcb0ceb)

**Comment by eyworldwide** [comment]
可以看一下这篇文档中关于 remap material 的操作：https://galacean.antgroup.com/engine/docs/latest/cn/graphics-material-editor
